src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,71 @@
+package frc.team4909.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.Encoder;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+import java.lang.module.ModuleDescriptor.Requires;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.team4909.robot.subsystems.elevator.commands.SetElevatorPosition;
+import frc.team4909.robot.Robot;
+import frc.team4909.robot.RobotConstants;
+import frc.team4909.robot.operator.controllers.BionicF310;
+
+public class ElevatorSubsystem extends Subsystem{
+    WPI_VictorSPX VictorSPX1, VictorSPX2, VictorSPX3;
+    WPI_TalonSRX masterSRX;
+
+    public int holdingPosition;
+
+    public void ElevatorSubsytem(){
+
+        //Lift
+        masterSRX = new WPI_TalonSRX(RobotConstants.elevatorSRXID);  //master SRX
+        VictorSPX1 = new WPI_VictorSPX(RobotConstants.elevatorSPX1ID); //slave SPX 1
+        VictorSPX2 = new WPI_VictorSPX(RobotConstants.elevatorSPX2ID); //slave SPX 2
+        VictorSPX3 = new WPI_VictorSPX(RobotConstants.elevatorSPX3ID); //slave SPX 3
+
+        VictorSPX1.follow(masterSRX);  //All slaves will output the same value as the master SRX
+        VictorSPX3.follow(masterSRX);  
+        VictorSPX2.follow(masterSRX);
```

Comment:
I see it’s been merged now, but these ideally shouldn’t be uppercase per convention